classifier_and_summarizer/summarizer/summarizer_output.py

```diff
@@ -0,0 +1,41 @@
+''' This script defines the object structure
+for the output(s) of the summarizer and provides
+methods to access the data as required'''
+
+
+class StampPages:
+    def __init__(self):
+        self.stamp_pages = list()
+
+    def add_stamp_page(self, stamp_page):
+        self.stamp_pages.append(stamp_page)
+
+    def get_formatted_list(self):
+        formatted_list = list()
+        for stamp_page in self.stamp_pages:
+            formatted_list.append(stamp_page.get_formatted_dict())
+        return formatted_list
+
+
+class StampPage:
+    def __init__(
+            self,
+            media_index,
+            is_embedded_content,
+            overlay_title,
+            overlay_text,
+            overlay_font_style,
+            overlay_font_size,
+            stamp_position):
+        self.media_index = media_index
+        self.is_embedded_content = is_embedded_content
```

Comment:
Can you add a comment here clarifying that by embedded_content we refer to twitter or instagram posts etc. and the explanation for media_text as well (so that anyone who reads the code can easily understand what they are).

Reply:
Updated accordingly!